### ISSUE TITLE
Transparency layer to allow full color of images

### ISSUE THREAD

**Issue by fjmontes** [okviz/synoptic-panel-issues#15]
### The feature request

When a new SVG image is created the layer added to the image diminishes the color of the image. 

### Proposed solution

 It is possible to add a transparent layer.

**Comment by danieleperilli** [comment]
Sorry, this is not very clear. Can you provide some examples?

**Comment by RVolders** [comment]
Hi @danieleperilli 

Under the setting "Data Colors" there is no option to disable the "Default Color".
It would be nice to have a "Transparent" option so you can see what is behind the area.
Enabeling the "Borders" option does this a tiny bit. But is not transparent enough to see what's behind it.
So maybe implementing a transparancy slicer from 0 to 100 would solve this.

Kind regards

**Comment by danieleperilli** [comment]
Yes, it makes sense. Thanks for suggesting it. We will include in the next Pro version that we release in the next couple of months.

**Comment by danieleperilli** [comment]
The new version is out: [https://okviz.com/synoptic-panel](https://okviz.com/synoptic-panel/)
